src/components/NewGalaxyModal.js

```diff
@@ -88,6 +127,11 @@ export const NewGalaxyModal = _.flow(
             div({ style: { lineHeight: 1.5 } }, [
               div(['Please delete the cloud environment when finished; it will']),
               div(['continue to ', span({ style: { fontWeight: 600 } }, ['incur charges ']), 'if it keeps running.'])
+            ]),
+            div({ style: { ...styles.headerText, marginTop: '0.5rem' } }, ['Pause and auto-pause']),
+            div({ style: { lineHeight: 1.5 } }, [
+              div(['You can pause anything during the compute, but it will auto-paused when']),
```

Comment:
What does anything mean here? I think it would be good to be specific about what the user can pause, possibly listed out.

Reply:
oh also probably should be "will auto-pause when" for the correct tense, also "idle" doesn't need to be past tense I don't think.